src/cli/repl/server/messages/slice.ts

```diff
@@ -23,7 +23,7 @@ export const requestSliceMessage: MessageDefinition<SliceRequestMessage> = {
 		type:      Joi.string().valid('request-slice').required(),
 		id:        Joi.string().optional(),
 		filetoken: Joi.string().required(),
-		criterion: Joi.array().items(Joi.string().regex(/\d+:\d+|\d+@.*|\$\d+/)).min(0).required()
+		criterion: Joi.array().items(Joi.string().regex(/\d+:\d+|\d+@.*|\$(\d+|.+@root-\d+-\d+)/)).min(0).required()
```

Comment:
Not sure if we should narrow down the path of the ID. E.g. `/a/b/c.R` in `/a/b/c.R|root-1-1`. Currently, it is matched by `.+`. But we could also be a bit more specific as it could be a real path, or just "-inline-"